testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -633,570 +642,6 @@ def test_write_not_changing_ts(self, ref, universe, tmpdir):
                 assert_timestep_almost_equal(copy_ts, universe.trajectory.ts)
 
 
-class BaseTimestepTest(object):
```

Comment:
Just checking since I have yet to wake up properly - we are moving this because custom Timestep objects have now been removed right?

Reply:
Yeah we used to have many Timesteps and this was the base set of tests for them.  Now we've just got one Timestep so you don't need to generalise the tests like this, there's just a single testsuite for Timestep.